types/diff-match-patch/index.d.ts

```diff
@@ -1,21 +1,21 @@
-// Type definitions for diff-match-patch v1.0.0
-// Project: https://www.npmjs.com/package/diff-match-patch
-// Definitions by: Asana <https://asana.com>
+// Type definitions for node-diff-match-patch 1.0
+// Project: https://github.com/ForbesLindesay/diff-match-patch
+// Definitions by: Asana <https://github.com/asana>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
 
+export type Diff = [number, string];
 
-type Diff = [number, string];
-
-export declare class Patch {
+export class patch_obj {
     diffs: Diff[];
-    start1: number;
-    start2: number;
+    start1: number | null;
+    start2: number | null;
     length1: number;
     length2: number;
 }
 
-export declare class diff_match_patch {
-    static new(): diff_match_patch;
+export class diff_match_patch {
+    constructor();
```

Comment:
is this line actually necessary? can we remove it?

Reply:
It would be better, however I can't guess the original intension.  I'll do it.